src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/ValueToDuplicatesTransformerTest.php

```diff
@@ -70,7 +70,7 @@ public function testReverseTransformCompletelyEmpty()
             'c' => '',
         ];
 
-        $this->assertNull($this->transformer->reverseTransform($input));
+        $this->assertSame('', $this->transformer->reverseTransform($input));
```

Comment:
```suggestion
        $this->assertEmpty($this->transformer->reverseTransform($input));
```
?

Reply:
I prefer to be precise here. The fact that we were precise before made me spot that we had this test and reconsider if I really did a desired change.